docs/dev/batch/index.md

```diff
@@ -588,9 +588,9 @@ val output: DataSet[(Int, String, Double)] = input.sum(0).min(2)
       <td>
         <p>Returns the distinct elements of a data set. It removes the duplicate entries
         from the input DataSet, with respect to all fields of the elements, or a subset of fields.</p>
-      {% highlight scala %}
-         data.distinct()
-      {% endhighlight %}
+{% highlight scala %}
```

Comment:
In this file there's also some unnecessary whitespace in the Java Dataset Aggregate transformation.